app/routes/task_routes.py

```diff
@@ -1 +1,90 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, abort, make_response, Response
+from ..db import db
+from .route_utilities import validate_model, update_dict_to_database, add_dict_to_database, update_complete_at_to_database
+from app.models.task import Task
+from sqlalchemy import desc
+import datetime
+import requests
+import json
+import os
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+
+@tasks_bp.post("")
+def create_task():
+    request_body = request.get_json()
+    task = add_dict_to_database(Task, request_body)
+    return {"task": task.to_dict()}, 201
+
+
+@tasks_bp.get("")
+def get_all_tasks():
+    query = db.select(Task)
+    sort_param = request.args.get("sort", "asc")
+    if sort_param == "asc":
+        query = query.order_by(Task.title)
+    elif sort_param == "desc":
+        query = query.order_by(desc(Task.title))
+    else:
+        response = {"details": "Wrong parameter value"}
+        abort(make_response(response, 400))
```

Comment:
What if instead of sending back an error response with 400 status code, you assumed that if a client doesn't want to sort by asc or desc, then we'd have a fallback to order by id when no sort is requested?
```suggestion
    else:
        query = db.select(Task).order_by(Task.id)
```